airbyte/mcp/cloud_ops.py

```diff
@@ -26,6 +28,37 @@
 from airbyte.mcp._util import resolve_config, resolve_list_of_strings
 
 
+def _check_internal_admin_flag() -> str:
```

Comment:
Actually no. Just decouple this. The check operation can return true or false after comparing both values, optionally printing warnings for invalid input.

But a separte internal helper function can return the email - calling the check function first and then independently reading the email var if 'bool' is returned.